tests/test_vxlan_vtep_vni.rb

```diff
@@ -49,6 +49,8 @@ def test_create_with_existing
   end
 
   def test_vnis
+    skip('Platform does not support vnis') if VxlanVtepVni.vnis['nve1'].nil?
```

Comment:
Why are you only skipping this one test case.  Shouldn't the other test cases be skipped for unsupported platforms?